src/anomalib/utils/hpo/sweep.py

```diff
@@ -28,24 +25,17 @@ def __str__(self) -> str:
 
 
 def get_hpo_parser(
-    parser: ArgumentParser | LightningArgumentParser | None = None,
-) -> ArgumentParser | LightningArgumentParser:
+    parser: LightningArgumentParser | None = None,
+) -> LightningArgumentParser:
     """Get the HPO parser."""
     if parser is None:
-        parser = ArgumentParser()
-    parser.add_argument(
-        "--model",
-        type=str,
-        default="padim",
-        help="Name of the algorithm to train/test",
-        required=False,
-    )
-    parser.add_argument("--model_config", type=Path, required=False, help="Path to a model config file")
+        parser = LightningArgumentParser()
```

Comment:
is this indentation correct?

Reply:
The idea is to create the parser if it does not exist otherwise we add the arguments to the existing parser.

Reply:
argh git diff did not show the if statement. My bad, please ignore this one.